documentation/install-guide.html

```diff
@@ -191,158 +296,243 @@ <h3><a name="autosetup"></a>Autosetup</h3>
-deleting its JAR file (Openfire will automatically remove the plugin from memory and delete it's directory when you
-do so).
-    
-<h2><a name="plugins">Firewall</a></h2>
-    
-Server's administrator should open TCP port 5222 for incoming connections for XMPP clients to be able to connect to Openfire.
-Additionally an administrator can also open TCP 9090 (for http) and TCP 9091 (for https), if there is a need to remotely administrate
-Openfire connecting to its Admin Console. We recommend only using 9091 port as it is an encrypted connection. For server to server
-connections one should also open TCP 5269 port and for secure HTTP-BIND connections TCP 7443 port. Port number can be different, if
-the default configuration has been changed by an administrator. All ports used by Openfire can be found on the first page of 
-Admin Console in the Server Ports section.
-
-    </div>
+    &lt;/autosetup&gt;</code></pre>
+            </fieldset>
+
+            <p>
+                Then setup will be run on first launch, and the <code>&lt;autosetup/&gt;</code> section will be deleted from
+                <code>openfire.xml</code>
+            </p>
+
+            <p>
+                Note that autosetup provides fewer options to setup Openfire as compared to using the setup wizard.
+            </p>
+
+        </section>
+
+        <section id="tools">
+
+            <h3>Admin Console</h3>
 
-</div>
+            <p>
+                After completing the above steps, Openfire will be configured through, and you can use the web-based admin
+                console to administer the server. The URL should be the same as you used to setup the server unless you
+                changed the port during the setup.
+            </p>
+
+        </section>
+
+    </section>
+
+    <hr>
+
+    <section id="windows">
+
+        <h2>Running Openfire on Windows</h2>
+
+        <p>
+            Since 4.1.5 version Openfire automatically installs and runs the service (and opens the browser with the web
+            setup page). But you can also use the launcher, if you need to (service has to be stopped before running the
+            launcher). If you used the Openfire installer, a shortcut for starting the graphical launcher is provided in
+            your Start Menu. Otherwise, run openfire.exe in the <code>bin/</code> directory of your Openfire
+            installation. A button on the launcher allows you to automatically open your web browser to the correct URL
+            to finish setting up the server: <a href="http://127.0.0.1:9090">http://127.0.0.1:9090</a>
+        </p>
+
+        <section>
+
+            <h3>Windows Service</h3>
+
+            <figure class="inline-right">
+                <img src="images/windows_service.png" alt="Windows Service" width="557" height="223">
+                <figcaption>Windows Service commands in a console windows.</figcaption>
+            </figure>
```

Comment:
I'll try and source a replacement image for this

Reply:
Done, but I thought the background text was part of an old screenshot, but it isn't. It's a formatting problem on install-guide where the descriptions are being squished between the terms and the screenshot.

![image](https://user-images.githubusercontent.com/2117083/204386351-724058e8-1903-4bdf-908d-2d0c2a73061c.png)


I'd offer to fix it, but I'd have no clue where to begin. Sorry...

Reply:
Ah, thanks for updating that! I think that I've fixed the alignment-issue. Force-pushed the fix to your last commit.